timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -76,6 +76,48 @@ limitations under the License.
         </v-menu>
       </v-card>
 
+      <!-- Search History -->
+      <div class="mt-4">
+        <v-card v-show="showSearchHistory" outlined>
+          <v-toolbar dense flat>
+            <v-toolbar-title>Search history</v-toolbar-title>
+            <v-spacer></v-spacer>
+            <v-slider
+              v-model="zoomLevel"
+              thumb-label
+              ticks
+              append-icon="mdi-magnify-plus-outline"
+              prepend-icon="mdi-magnify-minus-outline"
+              min="0.1"
+              max="1"
+              step="0.1"
+              class="mt-6"
+            >
+              <template v-slot:thumb-label="{ value }"> {{ value * 100 }}% </template>
+            </v-slider>
+
+            <v-btn icon @click="showSearchHistory = false" class="ml-4">
+              <v-icon>mdi-close</v-icon>
+            </v-btn>
+          </v-toolbar>
+
+          <v-divider></v-divider>
+
+          <div
+            v-dragscroll
+            class="pa-md-4 no-scrollbars"
+            style="overflow: scroll; white-space: nowrap; max-height: 500px; min-height: 500px"
```

Comment:
Having min-height = max-height allows for now dynamic scaling of the history tee. Most trees I have seen in the past were pretty linear and had maybe 2-3 layers. Fixing the height to 500px makes the search history a solid big block with a lot of white space. I would recommend going with min 100 px.